gomock/callset.go

```diff
@@ -56,6 +67,11 @@ func (cs callSet) Add(call *Call) {
 	if call.exhausted() {
 		m = cs.exhausted
 	}
+	if cs.allowOverride {
+		delete(m, key)
```

Comment:
this is redundant. You're overwriting it with an empty slice in the line below this anyway, so this shouldn't be necessary.

Reply:
thanks, updated.